roles/kuryr_node/tasks/main.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Create CNI bin directory
+  file:
+    state: directory
+    path: "{{ cni_bin_dir }}"
+    mode: 0755
+    owner: root
+    group: root
+    recurse: yes
+
+- name: Download and extract standard CNI plugins
+  become: yes
+  unarchive:
+    remote_src: True
+    src: "{{ cni_bin_url }}"
+    dest: "{{ cni_bin_dir }}"
+
+- name: Ensure CNI net.d exists
+  file:
+    path: /etc/cni/net.d
+    recurse: yes
+    state: directory
+
+- name: Install loopback CNI conf
+  become: yes
+  copy:
+    src: 99-loopback.conf
```

Comment:
Does the Kuryr CNI plugin write a file to /etc/cni/net.d that is asciibetically "earlier" than this one?  Otherwise Kube might find 99-loopback and never call your kuryr plugin.

In general with kube/openshift it's advised to not install any CNI .conf files by default if you know you'll be running your own CNI plugin.

Reply:
Yes, Kuryr Docker image is placing 10-kuryr.conf file.

I've did initial testing and it looks like everything is fine without 99-loopback.conf. Moreover Calico role isn't installing it. I guess it can be removed, but I'll confirm that with @celebdor.